sdks/java/core/src/main/java/org/apache/beam/sdk/values/PCollection.java

```diff
@@ -326,7 +326,8 @@ public IsBounded isBounded() {
    */
   private final TupleTag<?> tag = new TupleTag<>();
 
-  private PCollection(Pipeline p) {
+  @Internal
+  protected PCollection(Pipeline p) {
```

Comment:
If my reservation about `RawPTransform` is accurate, re-privatize.

If it is not, this should have associated Javadoc.